### PR TITLE
Worked on the Ui intergartion of rewards page

### PR DESCRIPTION
**Pull request by Sanjeev1308**
Changed the rewards containers logic because not able to find data
Worked on the data display, read the data from logic components and displayed with proper format

### CODE REVIEW

**Review by mohsenParvar — APPROVED** [review]

File: src/app/pages/Rewards/components/CurrencyReserve/CurrencyInfo.tsx [hunk 1]
```diff
@@ -54,9 +70,9 @@ const StyledInfo = styled("div")({
   padding: 20,
 });
 
-const IconImage = styled("img")({
-  width: "33px",
-});
+// const IconImage = styled("img")({
```
Comment (saito-sv) [inline src/app/pages/Rewards/components/CurrencyReserve/CurrencyInfo.tsx]: remove unused code.

File: src/app/pages/Rewards/selectors.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    createSelector(RewardsDomains.pools, (pools) => pools[key]),
+  rewardsPoolTokens: (key: string) =>
+    createSelector(RewardsDomains.pools, (pools) => pools[key].poolTokens),
+  rewardsPoolData: (key: string) =>
+    createSelector(RewardsDomains.pools, (pools) => {
+      if (key === Pools.AXIAL_JLP) {
+        return {
+          tokens: [
+            {
+              percent: "24.19%",
```
Comment (saito-sv) [inline src/app/pages/Rewards/selectors.ts]: lets not hardcode values, instead consider using a common file for static values.

**Review by saito-sv — APPROVED** [review]
LGTM, please address some minor comments.

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-task-sanjeev-rewards-integration.dev-app.axial.exchange

**Review by bigwampa — CHANGES_REQUESTED** [review]
Compound w/ Snowball button should link to the appropriate pool on the compound & earn section of snowball.
Screen Shot 2022-03-22 at 2 31 13 PM

Approval Checkbox should be visible.
Screen Shot 2022-03-22 at 2 33 49 PM

There is no slippage when depositing single assets into a rewards pool, please remove.
Screen Shot 2022-03-22 at 2 34 27 PM

Advanced options should toggle open and closed. They should be closed by default.
Screen Shot 2022-03-22 at 2 34 33 PM

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-task-sanjeev-rewards-integration.dev-app.axial.exchange

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-task-sanjeev-rewards-integration.dev-app.axial.exchange

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-task-sanjeev-rewards-integration.dev-app.axial.exchange

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-task-sanjeev-rewards-integration.dev-app.axial.exchange

**Pull request merged**